internal/stoppropaganda/websites.go

```diff
@@ -400,114 +432,117 @@ func startWebsites() {
 	}()
 }
 
+func (ws *Website) ShouldRun() (shouldRun bool) {
+	shouldRun = true // true by default
+	ws.mux.Lock()
+	if ws.paused {
+		paused := time.Now().Before(ws.unpauseTime)
+		if !paused {
+			ws.paused = false
+		}
+		shouldRun = !paused
```

Comment:
You are not doing anything with this variable

Reply:
`shouldRun` is used as a return value